tests/e2e/specs/activate-and-setup/setup-wizard.test.js

```diff
@@ -1,35 +1,240 @@
+			await expect( page ).toMatchElement( 'button[disabled]', {
+				text: 'Continue',
+			} );
+		} );
+
+		it( 'allows selecting purposes', async () => {
+			await expect( page ).toClick( 'label', {
+				text: 'Promote your business',
+			} );
+			await expect( page ).toClick( 'label', { text: 'Other' } );
+			await expect( page ).toFill(
+				'.sensei-setup-wizard__textcontrol-other input',
+				'Other'
+			);
+
+			await expect( page ).toClick( 'button', { text: 'Continue' } );
+		} );
+
+		it( 'marks purpose step done and goes to features step', async () => {
+			await stepIsComplete( 'Purpose' );
+			await stepIsActive( 'Features' );
+			await expect( page ).toMatch(
+				'Enhance your online courses with these optional features.'
+			);
+		} );
+	} );
+
+	describe( 'Features step', () => {
+		beforeAll( async () => {
+			await AdminFlow.deactivatePlugin( 'sensei-media-attachments' );
+			await AdminFlow.deactivatePlugin( 'sensei-certificates' );
+			await openSetupWizard();
+		} );
+
+		it( 'opens when it is the active step', async () => {
+			await stepIsComplete( 'Purpose' );
+			await stepIsActive( 'Features' );
+			await expect( page ).toMatch(
+				'Enhance your online courses with these optional features.'
+			);
+		} );
+
+		it( 'allows selecting plugins', async () => {
+			await expect( page ).toClick( 'label', {
+				text: 'Sensei LMS Certificates',
+			} );
+		} );
+
+		it( 'confirms plugin installation', async () => {
+			await expect( page ).toClick( 'button', { text: 'Continue' } );
+			await expect( page ).toMatch(
+				'Would you like to install the following features now?'
+			);
+
+			await expect( page ).toMatchElement(
+				'.sensei-setup-wizard__features-confirmation-modal .woocommerce-list__item-title',
+				{
+					text: 'Sensei LMS Certificates',
+				}
+			);
+		} );
+		it( 'installs selected plugins', async () => {
+			await expect( page ).toClick( 'button', { text: 'Install now' } );
+
+			await expect( page ).toMatchElement(
+				'.woocommerce-list__item-title',
+				{
+					text: 'Sensei LMS Certificates — Installed',
+					timeout: 5000,
+				}
+			);
+			await expect( page ).toClick( 'button', { text: 'Continue' } );
+
+			await AdminFlow.goToPlugins();
+			expect(
+				await AdminFlow.findPlugin( 'sensei-certificates' )
+			).toBeTruthy();
+		} );
+
+		it( 'marks installed plugins as unavailable', async () => {
+			await openSetupWizard();
+
+			expect( page ).toClick( '.woocommerce-stepper__step', {
+				text: 'Features',
+			} );
+
+			await expect( page ).toMatchElement( '.status-installed', {
+				text: 'Sensei LMS Certificates — Installed',
+			} );
+		} );
+	} );
 
-		expect( deactivateLink ).toBeTruthy();
+	describe( 'Ready step', () => {
+		beforeEach( async () => {
+			await openSetupWizard();
+			await toClick( page, '.woocommerce-stepper__step', {
+				text: 'Ready',
+			} );
+		} );
+		it( 'is available if it is the active step', async () => {
```

Comment:
There are some tests that we have break line between and others not. Could we add a break line between all of them? So we keep it similar to the unit tests :)

Reply:
Cleaned it up in 74bd078